package.json

```diff
@@ -40,6 +40,8 @@
     "babelify": "^7.2.0",
     "bluebird": "^3.0.6",
     "browserify": "^12.0.1",
+    "lodash": "^3.10.1",
+    "tsify": "^0.13.1",
```

Comment:
Just did some testing and, due to the same dependency issue, I had to install the `babel-preset-es2015` package in my module (back/module/<name>) dirs. Would probably want to have that somewhere globally.